-- Adminer 4.7.7 PostgreSQL dump

INSERT INTO "plugins" ("id", "name", "category_id", "description", "github_url", "language_id", "created_at", "updated_at") VALUES
(1,	'AbundCSV2NOA',	1,	'Convert CSV file of abundances to NOA for Cytoscape',	'https://github.com/movingpictures83/AbundCSV2NOA',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(2,	'BIOM2CSV',	1,	'Convert BIOM file to CSVA',	'https://github.com/verociraptor/BIOM2CSV',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(3,	'ClusterCSV2NOA',	1,	'Convert CSV File Of Clusters to NOA',	'https://github.com/movingpictures83/ClusterCSV2NOA',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(4,	'CountTableProcessing',	1,	'Converts Mothur Counts To Abundance CSV',	'https://github.com/movingpictures83/CountTableProcessing',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(5,	'CSVAppend',	1,	'Append a new row or column to a CSV file',	'https://github.com/movingpictures83/CSVAppend',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(6,	'CSV2EDA',	1,	'Converter from CSV to Cytoscape EDA file format',	'https://github.com/movingpictures83/CSV2EDA',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(7,	'CSV2GML',	1,	'CSV To GML Converter',	'https://github.com/movingpictures83/CSV2GML',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(8,	'CSV2PathwayTools',	1,	'Take a CSV file and perform lookups in PathwayTools (Karp et al, 2015)',	'https://github.com/movingpictures83/CSV2PathwayTools',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(9,	'CSV2Tab',	1,	'Converter from comma-separated to tab-delimited format',	'https://github.com/verociraptor/CSV2Tab',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(10,	'CSVExtract',	1,	'Extract rows from a CSV file with a specific value in a column',	'https://github.com/movingpictures83/CSVExtract',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(11,	'CSVFilter',	1,	'Remove Columns from a CSV file with All Zero Values',	'https://github.com/movingpictures83/CSVFilter',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(12,	'CSVIndex',	1,	'Index a CSV file over a range of columns',	'https://github.com/movingpictures83/CSVIndex',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(13,	'CSVLabel',	1,	'Label rows of a CSV file',	'https://github.com/movingpictures83/CSVLabel',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(14,	'CSVLast2First',	1,	'Move the last column of a CSV file to the first',	'https://github.com/movingpictures83/CSVLast2First',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(15,	'CSVMerge',	1,	'Merge Multiple CSV Files',	'https://github.com/movingpictures83/CSVMerge',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(16,	'CSVPad',	1,	'CSV Padding',	'https://github.com/movingpictures83/CSVPad',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(17,	'CSVRemove',	1,	'Remove a column from a CSV file',	'https://github.com/movingpictures83/CSVRemove',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(18,	'CSVScreen',	1,	'Remove all rows with a zero or non-zero value in a specific column',	'https://github.com/movingpictures83/CSVScreen',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(19,	'CSVScreenTax',	1,	'Remove all rows with a zero or non-zero abundance for a specific taxon (any phylogenetic tree level)',	'https://github.com/movingpictures83/CSVScreenTax',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(20,	'CSVTrim',	1,	'Trim a CSV file to a specific set of columns',	'https://github.com/movingpictures83/CSVTrim',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(21,	'FASTA2FASTQ',	1,	'Convert from FASTA to FASTQ format',	'https://github.com/movingpictures83/FASTA2FASTQ',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(22,	'FASTQ2FASTA',	1,	'Convert from FASTQ to FASTA format',	'https://github.com/movingpictures83/FASTQ2FASTA',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(23,	'FilterPathway',	1,	'Take a CSV and find all pathways present in PathwayTools (Karp et al, 2015)',	'https://github.com/movingpictures83/FilterPathway',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(24,	'GML2CSV',	1,	'GML To CSV Converter',	'https://github.com/verociraptor/GML2CSV',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(25,	'LabelDups',	1,	'Take a CSV file and label duplicate entries',	'https://github.com/movingpictures83/LabelDups',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(26,	'PathwayFilter',	1,	'Take a CSV and remove all connections with no corresponding pathway in PathwayTools (Karp et al, 2015)',	'https://github.com/movingpictures83/PathwayFilter',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(27,	'PathwayTools2HMDB',	1,	'Map PathwayTools (Karp et al, 2015) identifiers to HMDB',	'https://github.com/movingpictures83/PathwayTools2HMDB',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(28,	'PCL2CSV',	1,	'PCL To CSV (File Conversion)',	'https://github.com/movingpictures83/PCL2CSV',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(29,	'SIF2CSV',	1,	'SIF To CSV (Interactions to Correlations)',	'https://github.com/movingpictures83/SIF2CSV',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(30,	'SplitBIOM',	1,	'Take a BIOM File and split it into each taxonomic tree level',	'https://github.com/movingpictures83/SplitBIOM',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(31,	'Tab2CSV',	1,	'Converter from tab-delimited to comma-separated format',	'https://github.com/verociraptor/Tab2CSV',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(32,	'Tab2GML',	1,	'Converter from tab-delimited to GML format',	'https://github.com/movingpictures83/Tab2GML',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(33,	'Abundance',	2,	'Average abundance of an entity in a sample set.',	'https://github.com/movingpictures83/Abundance',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(34,	'AlphaDiversity',	2,	'Alpha-diversity using Qiime (Caporaso et al, 2010)',	'https://github.com/movingpictures83/AlphaDiversity',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(35,	'Atlas',	2,	'Plot the abundance of a taxon over samples, colored by category',	'https://github.com/movingpictures83/Atlas',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(36,	'ANOSIM',	2,	'Analysis of Similarities (ANOSIM) (Clarke, 1993)',	'https://github.com/verociraptor/ANOSIM',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(37,	'Bar',	2,	'Bar graph of OTU abundances',	'https://github.com/movingpictures83/Bar',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(38,	'BetaDiversity',	2,	'Beta-diversity using Qiime (Caporaso et al, 2010)',	'https://github.com/movingpictures83/BetaDiversity',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(39,	'BoxPlot',	2,	'Produce a boxplot',	'https://github.com/movingpictures83/BoxPlot',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(40,	'CalcMeanStd',	2,	'Calculates Mean and Standard Deviation',	'https://github.com/movingpictures83/CalcMeanStd',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(41,	'Core',	2,	'Core taxa community statistics',	'https://github.com/movingpictures83/Core',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(42,	'CSV2Dot',	2,	'Take a Network in CSV Format and Visualize in Dot (Gansner et al, 2015)',	'https://github.com/movingpictures83/CSV2Dot',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(43,	'CSVAvgDeg',	2,	'Compute the average node degree of a network.',	'https://github.com/verociraptor/CSVAvgDeg',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(44,	'CSVAvgEdgeWeight',	2,	'Compute the average edge weight of a network.',	'https://github.com/verociraptor/CSVAvgEdgeWeight',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(45,	'CSVDup',	2,	'List duplicate row and column entries in a CSV file.',	'https://github.com/movingpictures83/CSVDup',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(46,	'CSVMax',	2,	'Determines Maximum Count Within Samples',	'https://github.com/movingpictures83/CSVMax',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(47,	'CSVNegOnly',	2,	'Output negative edges.',	'https://github.com/movingpictures83/CSVNegOnly',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(48,	'CSVOverlap',	2,	'Vector overlap',	'https://github.com/movingpictures83/CSVOverlap',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(49,	'CytoViz',	2,	'Cytoscape Visualizer',	'https://github.com/movingpictures83/CytoViz',	2,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(50,	'Delta',	2,	'Calculate differences in abundance (mean, std dev, etc.) between two samples',	'https://github.com/movingpictures83/Delta',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(51,	'Density',	2,	'Density plot (normalized abundance vs frequency) of a target taxon',	'https://github.com/movingpictures83/Density',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(52,	'DIABLO',	2,	'Generate DIABLO (Singh et al, 2019) plot',	'https://github.com/movingpictures83/DIABLO',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(53,	'Diversity',	2,	'Various diversity measurements',	'https://github.com/movingpictures83/Diversity',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(54,	'Dominance',	2,	'Dominance index (Locey and Lennon, 2016)',	'https://github.com/movingpictures83/Dominance',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(55,	'Evenness',	2,	'Evenness indices',	'https://github.com/movingpictures83/Evenness',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(56,	'Extract',	2,	'Extract sequences that include a target sequence',	'https://github.com/movingpictures83/Extract',	2,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(57,	'Gini',	2,	'Gini index (Gini, 1912)',	'https://github.com/movingpictures83/Gini',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(58,	'GraPhlAn',	2,	'Visualize phylogenetic tree with GraPhlAn (Asnicar et al, 2013)',	'https://github.com/movingpictures83/GraPhlAn',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(59,	'Gnuplot',	2,	'Plot data using Gnuplot (Williams and Kelley, 2011)',	'https://github.com/movingpictures83/Gnuplot',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(60,	'Heatmap',	2,	'Render a heatmap',	'https://github.com/movingpictures83/Heatmap',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(61,	'HIVE',	2,	'HIVE (Hanson, 2011) plots',	'https://github.com/movingpictures83/HIVE',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(62,	'InverseSimpson',	2,	'Inverse Simpson Diversity Index',	'https://github.com/movingpictures83/InverseSimpson',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(63,	'Krona',	2,	'Krona plot',	'https://github.com/movingpictures83/Krona',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(64,	'Landscape',	2,	'Landscape plot of sample sets vs. first two principal components',	'https://github.com/movingpictures83/Landscape',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(65,	'LogModulo',	2,	'Log-modulo skew (Magurran and McGill, 2011)',	'https://github.com/movingpictures83/LogModulo',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(66,	'MM',	2,	'Partitions multi-omics data',	'https://github.com/movingpictures83/MM',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(67,	'MV',	2,	'Plot mean vs. variance',	'https://github.com/movingpictures83/MV',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(68,	'NetworkViz',	2,	'Network Visualizer (Generates Multiple Cytoscape Input Files)',	'https://github.com/movingpictures83/NetworkViz',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(69,	'NMDS',	2,	'Produce an OTU heatmap using Non-Multi-Dimensional Scaling (NMDS) (Kruskal, 1964)',	'https://github.com/movingpictures83/NMDS',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(70,	'OTUHeatmap',	2,	'Plot OTU abundances using a heatmap',	'https://github.com/movingpictures83/OTUHeatmap',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(71,	'OTUPlot',	2,	'Plot OTU abundances at multiple levels of the phylogenetic tree',	'https://github.com/movingpictures83/OTUPlot',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(72,	'Pie',	2,	'Phylum/class pie plot',	'https://github.com/movingpictures83/Pie',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(73,	'ProcessDesc',	2,	'Extract phylogenetic information from FASTA file',	'https://github.com/movingpictures83/ProcessDesc',	2,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(74,	'QGraph',	2,	'QGraph Visualizer',	'https://github.com/movingpictures83/QGraph',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(75,	'Rank',	2,	'Converts a set of counts to ranks in a sample',	'https://github.com/movingpictures83/Rank',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(76,	'Rare',	2,	'Rare taxa community statistics',	'https://github.com/movingpictures83/Rare',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(77,	'ReactionPathway',	2,	'ReactionPathway (Statistical Analysis of Metabolite Network and Reaction Pathway Data)',	'https://github.com/movingpictures83/ReactionPathway',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(78,	'Richness',	2,	'Sample set richness',	'https://github.com/movingpictures83/Richness',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(79,	'Sparsity',	2,	'Plot sparsity of a dataset',	'https://github.com/movingpictures83/Sparsity',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(80,	'Spread',	2,	'Plot spread of taxa abundances over all samples',	'https://github.com/movingpictures83/Spread',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(81,	'Statistics',	2,	'Take a set of samples and output statistics (mean, standard deviation, etc.) on the most abundant taxa',	'https://github.com/movingpictures83/Statistics',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(82,	'Summarize',	2,	'Output a summary of a sample set to the screen',	'https://github.com/movingpictures83/Summarize',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(83,	'Tree',	2,	'PluMA plugin to produce a phylogenetic tree for taxa, decorated by sample set',	'https://github.com/movingpictures83/Tree',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(84,	'TriadCounter',	2,	'Counts the number of stable and unstable triads (Easley and Kleinberg, 2010) in a social network',	'https://github.com/movingpictures83/TriadCounter',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(85,	'TTestDependent',	2,	'Run the T-Test on dependent samples',	'https://github.com/verociraptor/TTestDependent',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(86,	'TTestIndependent',	2,	'Run the T-Test on independent data',	'https://github.com/verociraptor/TTestIndependent',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(87,	'Variance',	2,	'Variance Calculator',	'https://github.com/movingpictures83/Variance',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(88,	'Violin',	2,	'Violin plot',	'https://github.com/movingpictures83/Violin',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(89,	'ASV2Taxon',	3,	'Assign unique taxa to Amplicon Sequence Variants (ASVs) (Callahan, 2017)',	'https://github.com/movingpictures83/ASV2Taxon',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(90,	'CLR',	3,	'Center Log Ratio (CLR) transformation',	'https://github.com/movingpictures83/CLR',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(91,	'Clusterize',	3,	'Remove Edges Between Nodes in Different Clusters',	'https://github.com/movingpictures83/Clusterize',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(92,	'CSVJoinSplit',	3,	'Generate taxonomically classified taxa counts from original counts, metadata and taxonomic tree',	'https://github.com/movingpictures83/CSVJoinSplit',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(93,	'CSVMapRange',	3,	'Map Range of One CSV File To Another (Useful in Multi-Omics)',	'https://github.com/movingpictures83/CSVMapRange',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(94,	'CSVMult',	3,	'Multiply a CSV file by a constant',	'https://github.com/movingpictures83/CSVMult',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(95,	'CSVNeg2Pos',	3,	'Map [-1, 1] edge weight range to [0, 2]',	'https://github.com/movingpictures83/CSVNeg2Pos',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(96,	'CSVNeg2Zero',	3,	'Change Negative Edges To Zero',	'https://github.com/movingpictures83/CSVNeg2Zero',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(97,	'CSVNormalize',	3,	'CSV Normalization',	'https://github.com/movingpictures83/CSVNormalize',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(98,	'CSVPathways',	3,	'Take a CSV and lookup all metabolic pathways involving internal taxa or metabolites',	'https://github.com/movingpictures83/CSVPathways',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(99,	'CSVPool',	3,	'Take a CSV and pool together duplicate rows/columns',	'https://github.com/movingpictures83/CSVPool',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(100,	'CSVRename',	3,	'Rename rows and/or columns of a CSV',	'https://github.com/movingpictures83/CSVRename',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(101,	'CSVScale',	3,	'Scale CSV Values to Have a Median of One',	'https://github.com/movingpictures83/CSVScale',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(102,	'CSVSort',	3,	'Sort a CSV file by row',	'https://github.com/movingpictures83/CSVSort',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(103,	'CSVTranspose',	3,	'Transpose a Matrix in a CSV File',	'https://github.com/movingpictures83/CSVTranspose',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(104,	'CSVUnNormalize',	3,	'CSV Reverse Normalization',	'https://github.com/movingpictures83/CSVUnNormalize',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(105,	'CSVZero2Min',	3,	'Change All Zeroes in a CSV to the Minimum Value',	'https://github.com/movingpictures83/CSVZero2Min',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(106,	'DeMUX',	3,	'Demultiplex (DeMUX) a set of sequences into groups given their barcodes (Caporaso et al, 2010)',	'https://github.com/movingpictures83/DeMUX',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(107,	'DeReplicate',	3,	'Remove replicate sequences',	'https://github.com/movingpictures83/DeReplicate',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(108,	'FilterMicrobes',	3,	'Filter out connections between metagenomic data based on underlying metabolic networks',	'https://github.com/movingpictures83/FilterMicrobes',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(109,	'Hellinger',	3,	'Hellinger transform (Hellinger et al, 1914)',	'https://github.com/movingpictures83/Hellinger',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(110,	'Log10',	3,	'Transform data by log base 10',	'https://github.com/movingpictures83/Log10',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(111,	'Log10P',	3,	'Transform data by log(1+x)',	'https://github.com/movingpictures83/Log10P',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(112,	'LogTransform',	3,	'Log-transform data',	'https://github.com/movingpictures83/LogTransform',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(113,	'Map2Positive',	3,	'Map All Edges To Positive Range',	'https://github.com/movingpictures83/Map2Positive',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(114,	'MetabNorm',	3,	'Mixed-Mode (Cohort and Batch) Metabolite Normalization (Jauhiainen et al, 2014)',	'https://github.com/movingpictures83/MetabNorm',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(115,	'NetworkDeconvolution',	3,	'Network deconvolution algorithm (Feizi et al, 2013) to determine direct network dependencies.',	'https://github.com/movingpictures83/NetworkDeconvolution',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(116,	'NetworkNoise',	3,	'Add random noise to a signed and weighted network.',	'https://github.com/movingpictures83/NetworkNoise',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(117,	'NormScoreTransform',	3,	'Normal Probability Values',	'https://github.com/movingpictures83/NormScoreTransform',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(118,	'Oksanen',	3,	'Oksanen transformation (Oksanen, 1983)',	'https://github.com/movingpictures83/Oksanen',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(119,	'OTUSummary',	3,	'Summarize OTUs at multiple levels of the phylogenetic tree',	'https://github.com/movingpictures83/OTUSummary',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(120,	'QuantumDenoiser',	3,	'Remove noisy fluctations from CKMeans quanta',	'https://github.com/movingpictures83/QuantumDenoiser',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(121,	'Rarefy',	3,	'Rarefying, ad hoc (McMurdie and Holmes, 2014)',	'https://github.com/movingpictures83/Rarefy',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(122,	'RemoveNegative',	3,	'Remove Negative Edges',	'https://github.com/movingpictures83/RemoveNegative',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(123,	'RemoveZero',	3,	'Remove zero-abundant entities',	'https://github.com/movingpictures83/RemoveZero',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(124,	'SplitTableProcessing',	3,	'Take a table of counts from Mothur and split into multiple counts classified by taxonomy',	'https://github.com/movingpictures83/SplitTableProcessing',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(125,	'Standardize',	3,	'Transform to standard distribution',	'https://github.com/movingpictures83/Standardize',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(126,	'TaxonomySplit',	3,	'Take a table of abundances and make multiple, one for each level of the taxonomic tree',	'https://github.com/movingpictures83/TaxonomySplit',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(127,	'Threshold',	3,	'Remove all observables from a dataset with a low frequency of occurrence',	'https://github.com/movingpictures83/Threshold',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(128,	'Transform',	3,	'Shift composition (Egozcue and Pawlowski-Glahn, 2016)',	'https://github.com/movingpictures83/Transform',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(129,	'UNOISE',	3,	'Denoise sequences (Edgar et al, 2016)',	'https://github.com/movingpictures83/UNOISE',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(130,	'ValidateMapping',	3,	'Validate mapping file with Qiime (Caporaso et al, 2010)',	'https://github.com/movingpictures83/ValidateMapping',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(131,	'Z',	3,	'Z-transformation',	'https://github.com/movingpictures83/Z',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(132,	'ANOVA',	4,	'ANalysis Of VAriance (ANOVA), estimates differences between sample groups (Kaufman and Scheren, 2014).',	'https://github.com/verociraptor/ANOVA',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(133,	'BHTSNE',	4,	'Barnes-Hut version of tSNE (Laczny et al, 2014)',	'https://github.com/movingpictures83/BHTSNE',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(134,	'BinomialDeviance',	4,	'Binomial Deviance (McArdle and Anderson, 2001)',	'https://github.com/movingpictures83/BinomialDeviance',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(135,	'Bray',	4,	'Bray-Curtis (Bray and Curtis, 1957)',	'https://github.com/movingpictures83/Bray',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(136,	'Canberra',	4,	'Canberra Distance (Lance and Williams, 1966)',	'https://github.com/movingpictures83/Canberra',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(137,	'Chao',	4,	'Chao&apos;s Method (Chao, 2005)',	'https://github.com/movingpictures83/Chao',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(138,	'DESeq',	4,	'Differential Gene Expression Analysis Based on the Negative Binomial Distribution (Love et al, 2014)',	'https://github.com/movingpictures83/DESeq',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(139,	'DPCoA',	4,	'Double Principal Component Analysis (Pavoine et al, 2004)',	'https://github.com/movingpictures83/DPCoA',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(140,	'EuclideanDistance',	4,	'Euclidean Distance',	'https://github.com/movingpictures83/EuclideanDistance',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(141,	'FeatureSelection',	4,	'Determine the most distinguishing features for case vs control',	'https://github.com/movingpictures83/FeatureSelection',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(142,	'Gower',	4,	'Gower Index (Gower, 1971)',	'https://github.com/movingpictures83/Gower',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(143,	'Horn',	4,	'Horn&apos;s Overlap Index (Horn, 1966)',	'https://github.com/movingpictures83/Horn',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(144,	'IPCA',	4,	'Independent Principle Component Analysis (Yao et al, 2012)',	'https://github.com/movingpictures83/IPCA',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(145,	'Jaccard',	4,	'Jaccard Index (Jaccard, 1912)',	'https://github.com/movingpictures83/Jaccard',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(146,	'JSD',	4,	'Jensen-Shannon Divergence (Fuglede et al, 2004)',	'https://github.com/movingpictures83/JSD',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(147,	'Kulczynski',	4,	'Kulczynski Similarity (Kulczynski, 1927)',	'https://github.com/movingpictures83/Kulczynski',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(148,	'LEfSe',	4,	'Linear discriminant analysis Effect Size (LEfSe) (Segata et al, 2011)',	'https://github.com/movingpictures83/LEfSe',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(149,	'LMS',	4,	'Least Mean Squares method for regression analysis (Widrow and Hoff, 1960)',	'https://github.com/movingpictures83/LMS',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(150,	'Manhattan',	4,	'Manhattan Distance',	'https://github.com/movingpictures83/Manhattan',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(151,	'Morisita',	4,	'Morisita Overlap (Morisita, 1959)',	'https://github.com/movingpictures83/Morisita',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(152,	'Mountford',	4,	'Mountford Dissimilariy Index (Mountford, 1962)',	'https://github.com/movingpictures83/Mountford',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(153,	'MT',	4,	'Multiple Testing of Taxa Abundances According to Sample Category (McMurdie, 2020) (',	'https://github.com/movingpictures83/MT',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(154,	'NetRep',	4,	'NetRep algorithm (Ritchie et al, 2016) for detecting amount of preservation between two datasets',	'https://github.com/movingpictures83/NetRep',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(155,	'PCA',	4,	'Principle Component Analysis (Pearson, 1901)',	'https://github.com/movingpictures83/PCA',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(156,	'PCoA',	4,	'Principle Coordinate Analysis (Gower, 1966)',	'https://github.com/movingpictures83/PCoA',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(157,	'PERMANOVA',	4,	'PERmanent Multivariate ANalysis Of VAriance (PERMANOVA) (Anderson, 2001)',	'https://github.com/verociraptor/PERMANOVA',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(158,	'PLSDA',	4,	'Partial Least Squares Discriminant Analysis (PLS-DA, Stahle and Wold 1987)',	'https://github.com/movingpictures83/PLSDA',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(159,	'undefined',	4,	'Partial Least Squares Discriminant Analysis (PLS-DA, Stahle and Wold 1987) for multiple datasets',	'undefined',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(160,	'Raup',	4,	'Raup-Crick (Raup and Crick, 1979)',	'https://github.com/movingpictures83/Raup',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(161,	'SpNMF',	4,	'Supervised Non-Negative Matrix Factorization (Cai et al, 2017)',	'https://github.com/movingpictures83/SpNMF',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(162,	'tSNE',	4,	't-Distributed Stochastic Neighbor Embedding (Van der Maaten et al, 2008)',	'https://github.com/movingpictures83/tSNE',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(163,	'UniFrac',	4,	'UniFrac distance (Lozupone and Knight, 2005)',	'https://github.com/movingpictures83/UniFrac',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(164,	'WeightedUniFrac',	4,	'Weighted UniFrac distance (Lozupone et al, 2007)',	'https://github.com/movingpictures83/WeightedUniFrac',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(165,	'CCA',	5,	'Canonical Correlation Analysis (Gonzalez et al, 2008)',	'https://github.com/movingpictures83/CCA',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(166,	'CoNet',	5,	'Infer abundance matrix from interactions using CoNet (Faust et al, 2012)',	'https://github.com/movingpictures83/CoNet',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(167,	'DistanceCorrelation',	5,	'Distance Correlation (Szekely, 2005)',	'https://github.com/movingpictures83/DistanceCorrelation',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(168,	'Ensemble',	5,	'Ensemble Correlation Computation (Weiss, 2016)',	'https://github.com/movingpictures83/Ensemble',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(169,	'FunChiSq',	5,	'Functional Chi-Squared Dependency Computation (Zhang and Song, 2013)',	'https://github.com/movingpictures83/FunChiSq',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(170,	'Kendall',	5,	'Kendall Correlation (Kendall, 1970)',	'https://github.com/movingpictures83/Kendall',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(171,	'KS',	5,	'Kolmogorov-Smirnov Test Statistic (Kolmogorov, 1933)',	'https://github.com/movingpictures83/KS',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(172,	'LSA',	5,	'Local Similarity Analysis (Ruan, 2006)',	'https://github.com/movingpictures83/LSA',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(173,	'MIC',	5,	'Maximal Information Coefficient (Reshef, 2011)',	'https://github.com/movingpictures83/MIC',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(174,	'MPLasso',	5,	'MPLasso (Lo and Marculescu, 2017)',	'https://github.com/movingpictures83/MPLasso',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(175,	'Pearson',	5,	'Pearson Correlation (Pearson, 1896)',	'https://github.com/movingpictures83/Pearson',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(176,	'Phi',	5,	'Proportionatlity metric (Lovell et al, 2015) for relative data',	'https://github.com/movingpictures83/Phi',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(177,	'PhiLR',	5,	'Phylogenetic Isometric Log-Ratio Transform (Silverman et al, 2017)',	'https://github.com/movingpictures83/PhiLR',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(178,	'RevEcoR',	5,	'Predict ecological relationships between two organisms based on metabolic networks using RevEcoR (Cao et al, 2016)',	'https://github.com/movingpictures83/RevEcoR',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(179,	'SparCC',	5,	'SparCC Correlation (Friedman and Alm, 2012)',	'https://github.com/movingpictures83/SparCC',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(180,	'SparCCGPU',	5,	'SparCC on the GPU (Alonso, Escobar, Panoff and Suarez, 2017)',	'https://github.com/movingpictures83/SparCCGPU',	5,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(181,	'Spearman',	5,	'Spearman Correlation (Spearman, 1904)',	'https://github.com/movingpictures83/Spearman',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(182,	'SpearmanMulti',	5,	'Spearman Correlations Over Multiple Data Sets and a Single Sample Set (Useful for Multi-Omics)',	'https://github.com/movingpictures83/SpearmanMulti',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(183,	'SpearmanTies',	5,	'Spearman, and Can Handle Rank Ties',	'https://github.com/movingpictures83/SpearmanTies',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(184,	'SpearmanTiesMulti',	5,	'Hybrid Functionality of SpearmanTies and SpearmanMulti',	'https://github.com/movingpictures83/SpearmanTiesMulti',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(185,	'SPIECEASI',	5,	'SPIEC-EASI (Kurtz et al, 2015)',	'https://github.com/movingpictures83/SPIECEASI',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(186,	'ATria',	6,	'Ablatio Triadum (ATria) Centrality (Cickovski et al, 2017)',	'https://github.com/movingpictures83/ATria',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(187,	'BiasedPageRank',	6,	'Edge-Weighted Personalized Page Rank (Xie et al, 2015)',	'https://github.com/movingpictures83/BiasedPageRank',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(188,	'Centroid',	6,	'Centroid algorithm (Wuchty and Stadler, 2003)',	'https://github.com/movingpictures83/Centroid',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(189,	'ClusterLeaders',	6,	'Compute leader nodes of network clusters',	'https://github.com/movingpictures83/ClusterLeaders',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(190,	'Conquests',	6,	'Conquests algorithm (Laniau et al, 2017) for Phenotypically Essential Metabolites',	'https://github.com/movingpictures83/Conquests',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(191,	'Degree',	6,	'Degree Centrality (Freeman, 1979)',	'https://github.com/movingpictures83/Degree',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(192,	'Exponential',	6,	'Exponential Centality (Benzi and Klymko, 2013)',	'https://github.com/movingpictures83/Exponential',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(193,	'GPUATria',	6,	'GPUATria (ATria, on the GPU)',	'https://github.com/movingpictures83/GPUATria',	5,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(194,	'IndVal',	6,	'Indicator species for each cluster (Roberts, 2007)',	'https://github.com/movingpictures83/IndVal',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(195,	'Katz',	6,	'Katz Prestige Centrality (Katz, 1953)',	'https://github.com/movingpictures83/Katz',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(196,	'MATria',	6,	'Multiple Ablatio Triadium (Cickovski et al, 2017)',	'https://github.com/movingpictures83/MATria',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(197,	'MWM',	6,	'Computes edge centralities, with the help of Maximum Weighted Matching (Galil, 1986)',	'https://github.com/movingpictures83/MWM',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(198,	'NetCooperate',	6,	'NetCooperate Algorithm for Key Metabolites (Levy et al, 2015)',	'https://github.com/movingpictures83/NetCooperate',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(199,	'PageRank',	6,	'Page Rank Centrality (Page, 1999)',	'https://github.com/movingpictures83/PageRank',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(200,	'PyATria',	6,	'ATria, in Python',	'https://github.com/movingpictures83/PyATria',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(201,	'RWRMH',	6,	'Random Walk with Restart on Multiplex and Heterogeneous biological networks (Valdeolivas et al, 2018)',	'https://github.com/movingpictures83/RWRMH',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(202,	'AffinityPropagation',	7,	'Affinity Propagation (Frey and Dueck, 2007)',	'https://github.com/movingpictures83/AffinityPropagation',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(203,	'CDHit',	7,	'CD-Hit (Li and Godzik, 2006)',	'https://github.com/movingpictures83/CDHit',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(204,	'ClusterCount',	7,	'Number of clusters in a network above a threshold in size',	'https://github.com/movingpictures83/ClusterCount',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(205,	'ClusterMeans',	7,	'Mean edge weight between all clusters of a network',	'https://github.com/movingpictures83/ClusterMeans',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(206,	'ClusterPathways',	7,	'Given a set of clusters of taxa, find all metabolic pathways shared by node pairs within the cluster',	'https://github.com/movingpictures83/ClusterPathways',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(207,	'ClusterStd',	7,	'Edge standard deviation between all clusters of a network',	'https://github.com/movingpictures83/ClusterStd',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(208,	'Community',	7,	'Information on the local community of a target.',	'https://github.com/movingpictures83/Community',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(209,	'CommunityDetection',	7,	'Girvan-Newman algorithm for community detection in a network (Girvan and Newman, 2002)',	'https://github.com/movingpictures83/CommunityDetection',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(210,	'CKMeans',	7,	'CKMeans (Wang and Song, 2011)',	'https://github.com/movingpictures83/CKMeans',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(211,	'DeNovoOTU',	7,	'De Novo OTU Selection (Caporaso et al, 2010)',	'https://github.com/movingpictures83/DeNovoOTU',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(212,	'Enterotypes',	7,	'Find enterotypes within a set of samples (Arumugam et al, 2011)',	'https://github.com/movingpictures83/Enterotypes',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(213,	'KMeans',	7,	'KMeans clustering (MacQueen, 1967)',	'https://github.com/movingpictures83/KMeans',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(214,	'LinClust',	7,	'Linear-Time Clustering (Steinegger and Soding, 2018)',	'https://github.com/movingpictures83/LinClust',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(215,	'Louvain',	7,	'Louvain Clustering (Blondel et al, 2008)',	'https://github.com/movingpictures83/Louvain',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(216,	'MCL',	7,	'Markov Clustering (van Dongen, 2000)',	'https://github.com/movingpictures83/MCL',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(217,	'MClust',	7,	'MClust (Scrucca et al, 2016)',	'https://github.com/movingpictures83/MClust',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(218,	'ModularityMaximization',	7,	'Modularity Maximization Algorithm For Clustering Level',	'https://github.com/movingpictures83/ModularityMaximization',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(219,	'PAM',	7,	'Partition Around Medoids (PAM) clustering (Kaufman and Rousseeuw, 1987)',	'https://github.com/movingpictures83/PAM',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(220,	'PrefixSuffix',	7,	'Prefix-Suffix algorithm of Qiime (Edgar et al, 2010)',	'https://github.com/movingpictures83/PrefixSuffix',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(221,	'PyNAST',	7,	'PyNAST clustering (Caporaso et al, 2010)',	'https://github.com/movingpictures83/PyNAST',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(222,	'RivalClub',	7,	'Find rival clubs in a network (Fernandez et al, 2015)',	'https://github.com/movingpictures83/RivalClub',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(223,	'SCIMM',	7,	'Sequence Clustering with Interpolated Markov Models (Kelley and Salzberg, 2010)',	'https://github.com/movingpictures83/SCIMM',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(224,	'SIMLR',	7,	'SIMLR (Wang et al, 2017)',	'https://github.com/movingpictures83/SIMLR',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(225,	'SortMerna',	7,	'SortMerna (Kopylova et al, 2014)',	'https://github.com/movingpictures83/SortMerna',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(226,	'Spectral',	7,	'Spectral Clustering (Meila and Shi, 2001)',	'https://github.com/movingpictures83/Spectral',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(227,	'SumaClust',	7,	'SumaClust (Mercier et al, 2014)',	'https://github.com/movingpictures83/SumaClust',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(228,	'SWARM',	7,	'SWARM (Mahe et al, 2014)',	'https://github.com/movingpictures83/SWARM',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(229,	'Tightness',	7,	'Various tightness measurements for a set of network clusters',	'https://github.com/movingpictures83/Tightness',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(230,	'TightnessStatistics',	7,	'Mean and standard deviation of cluster tightness',	'https://github.com/movingpictures83/TightnessStatistics',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(231,	'Trie',	7,	'Trie algorithm of Qiime (Edgar et al, 2010)',	'https://github.com/movingpictures83/Trie',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(232,	'UClust',	7,	'UClust (Edgar et al, 2010)',	'https://github.com/movingpictures83/UClust',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(233,	'UPARSE',	7,	'UPARSE (Edgar, 2013) algorithm',	'https://github.com/movingpictures83/UPARSE',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(234,	'USearch',	7,	'USearch (Edgar et al, 2010), adds filtering steps to UClust',	'https://github.com/movingpictures83/USearch',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(235,	'AutoCorrelation',	8,	'Autocorrelation Function Estimate',	'https://github.com/movingpictures83/AutoCorrelation',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(236,	'CCM',	8,	'Convergent Cross Mapping (Sugihara et al, 2012)',	'https://github.com/movingpictures83/CCM',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(237,	'CrossCorrelation',	8,	'Time-Series Cross-Correlation Values',	'https://github.com/movingpictures83/CrossCorrelation',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(238,	'CubicSpline',	8,	'Cubic Spline Interpolation',	'https://github.com/movingpictures83/CubicSpline',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(239,	'CumulativeClassifier',	8,	'Classify a set of samples by a set of features, at multiple timepoints',	'https://github.com/movingpictures83/CumulativeClassifier',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(240,	'CumulativeTime',	8,	'Choose the best features to use to classify a set of samples, at multiple timepoints.',	'https://github.com/movingpictures83/CumulativeTime',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(241,	'Detrend',	8,	'Detrend (Least-Squares Fit, Time Series Data)',	'https://github.com/movingpictures83/Detrend',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(242,	'DickeyFuller',	8,	'Dickey-Fuller Test, Time Series Data (Said and Dickey, 1984)',	'https://github.com/movingpictures83/DickeyFuller',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(243,	'Granger',	8,	'Granger Causality (Granger, 1969)',	'https://github.com/movingpictures83/Granger',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(244,	'LIMITS',	8,	'LIMITS algorithm for inferring correlations across time-series (Fisher and Mehta, 2014)',	'https://github.com/movingpictures83/LIMITS',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(245,	'NeuralNet',	8,	'Neural network for virus classification',	'https://github.com/movingpictures83/NeuralNet',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(246,	'RandomForest',	8,	'Random Forest classifier (Liaw and Wiener, 2002)',	'https://github.com/movingpictures83/RandomForest',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(247,	'ReplicationRates',	8,	'Bacterial Replication Rates (Brown et al, 2016)',	'https://github.com/movingpictures83/ReplicationRates',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(248,	'TimeSeries',	8,	'Form a time-series dataset by merging static and dynamic properties',	'https://github.com/movingpictures83/TimeSeries',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(249,	'TimeWarp',	8,	'Perform time warping to align biological data sets to same internal clock',	'https://github.com/movingpictures83/TimeWarp',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(250,	'AbundanceBin',	9,	'AbundanceBin binning tool (Wu et al, 2011) (Generated by PluGen)',	'https://github.com/movingpictures83/AbundanceBin',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(251,	'Atima',	9,	'Atima (Weick et al, 2000) (Generated by PluGen)',	'https://github.com/movingpictures83/Atima',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(252,	'BLSOM',	9,	'BLSOM unsupervised neural net (Nakao et al, 2013) (Generated by PluGen)',	'https://github.com/movingpictures83/BLSOM',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(253,	'Bowtie',	9,	'Bowtie sequence alignment and analysis (Langmead et al, 2009) (Generated by PluGen)',	'https://github.com/movingpictures83/Bowtie',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(254,	'Cytoscape',	9,	'Cytoscape Visualizer (Shannon et al, 2003) (Generated by PluGen)',	'https://github.com/movingpictures83/Cytoscape',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(255,	'DIAMOND',	9,	'DIAMOND  (Buchfink et al, 2014) (Generated by PluGen)',	'https://github.com/movingpictures83/DIAMOND',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(256,	'FastMRMR',	9,	'Fast Minimum Redundancy Maximum algorithm (Ramirez-Gallegos et al, 2016) for feature selection (Generated by PluGen)',	'https://github.com/movingpictures83/FastMRMR',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(257,	'FastQJoin',	9,	'FastQJoin (Blankenfeld et al, 2010) for joining paired-end reads (Generated by PluGen)',	'https://github.com/movingpictures83/FastQJoin',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(258,	'FragGeneScan',	9,	'FragGeneScan  (Rho et al, 2010) (Generated by PluGen)',	'https://github.com/movingpictures83/FragGeneScan',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(259,	'HMMER',	9,	'HMMER  (Finn et al, 2011) (Generated by PluGen)',	'https://github.com/movingpictures83/HMMER',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(260,	'HUMAnN',	9,	'HUMAnN (Franzosa et al, 2018) (Generated by PluGen)',	'https://github.com/movingpictures83/HUMAnN',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(261,	'Infomap',	9,	'Infomap (Edler et al., 2017) algorithm for community detection (Generated by PluGen)',	'https://github.com/movingpictures83/Infomap',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(262,	'IRep',	9,	'Calculate index of replication (Brown et al, 2016) (Generated by PluGen)',	'https://github.com/movingpictures83/IRep',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(263,	'KEGG',	9,	'Obtain all pathways for an organism in KEGG (Kanehisa and Goto, 2000) (Generated by PluGen)',	'https://github.com/movingpictures83/KEGG',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(264,	'MaxBin',	9,	'MaxBin (Wu et al, 2014) software for binning reads (Generated by PluGen)',	'https://github.com/movingpictures83/MaxBin',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(265,	'MEGAN',	9,	'MEGAN (Huson et al, 2007) (Generated by PluGen)',	'https://github.com/movingpictures83/MEGAN',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(266,	'MelitaSuite',	9,	'Melita (Jaric et al, 2013) suite of analysis pipelines for degenerate primers',	'https://github.com/movingpictures83/MelitaSuite',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(267,	'MetaBAT',	9,	'MetaBAT (Kang et al, 2015) (Generated by PluGen)',	'https://github.com/movingpictures83/MetaBAT',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(268,	'MetaCluster',	9,	'MetaCluster (Yang et al, 2010) (Generated by PluGen)',	'https://github.com/movingpictures83/MetaCluster',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(269,	'MetaPhlAn',	9,	'MetaPhlAn (Truong et al, 2015) (Generated by PluGen)',	'https://github.com/movingpictures83/MetaPhlAn',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(270,	'Mothur',	9,	'Mothur (Schloss, 2009) (Generated by PluGen)',	'https://github.com/movingpictures83/Mothur',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(271,	'MUSCLE',	9,	'MUltiple Sequence Comparison by Log-Expectation (MUSCLE) (Edgar et al, 2004) alignment tool',	'https://github.com/movingpictures83/MUSCLE',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(272,	'Network',	9,	'Represent samples as a network, with dissimilarity as edge weights',	'https://github.com/movingpictures83/Network',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(273,	'PathwayTools',	9,	'Get data from the PathwayTools (Karp et al, 2015) database',	'https://github.com/movingpictures83/PathwayTools',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(274,	'PICRUSt',	9,	'Phylogenetic Investigation of Communities by Reconstruction of Unobserved States (PICRUSt) (Lamille et al, 2013)',	'https://github.com/movingpictures83/PICRUSt',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(275,	'RDPClassify',	9,	'Classify sequences using the RDP (Wang et al, 2007) (Generated by PluGen)',	'https://github.com/movingpictures83/RDPClassify',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(276,	'Redbiom',	9,	'Query the Earth Microbiome Project (EMP, Thompson et al 2017) with Redbiom (McDonald et al, 2019) (Generated by PluGen)',	'https://github.com/movingpictures83/Redbiom',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(277,	'Synerclust',	9,	'Synerclust (Georgescu et al, 2018) orthologue clustering tool',	'https://github.com/movingpictures83/Synerclust',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(278,	'Tax2Tree',	9,	'Tax2Tree (McDonald et al, 2011) (Generated by PluGen)',	'https://github.com/movingpictures83/Tax2Tree',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(279,	'TetraESOM',	9,	'Tetramer frequencies, input to ESOM (Dick, 2009) (Generated by PluGen).',	'https://github.com/movingpictures83/TetraESOM',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(280,	'Amplicon',	99,	'Find organisms that produce a product when PCR is executed on a given primer pair',	'https://github.com/movingpictures83/Amplicon',	2,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(281,	'BarabasiAlbert',	99,	'Generate synthetic network (Barabasi and Albert, 1999)',	'https://github.com/movingpictures83/BarabasiAlbert',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(282,	'Caffe',	99,	'Deep learning through Convolutional Architecture for Fast Feature Embedding (Jia et al, 2014))',	'https://github.com/movingpictures83/Caffe',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(283,	'Classify',	99,	'Phylogenetic OTU Classifier',	'https://github.com/movingpictures83/Classify',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(284,	'ErdosRenyi',	99,	'Generate synthetic network (Erdos and Renyi, 1959)',	'https://github.com/movingpictures83/ErdosRenyi',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(285,	'FCI',	99,	'Fast Causal Inference (Spirtes et al, 1993)',	'https://github.com/movingpictures83/FCI',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(286,	'GeneUnify',	99,	'Unify gene expression values across multiple files',	'https://github.com/movingpictures83/GeneUnify',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(287,	'GPUKMM',	99,	'KMM, on the GPU',	'https://github.com/movingpictures83/GPUKMM',	5,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(288,	'KMM',	99,	'k-Markov Model for Metagenomics',	'https://github.com/movingpictures83/KMM',	3,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(289,	'LINGAM',	99,	'Linear Non-Gaussian Acyclic Model (LINGAM) for Causal Discovery (Shimizu et al, 2006)',	'https://github.com/movingpictures83/LINGAM',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(290,	'MakeCliques',	99,	'Make a Network of Cliques (Python)',	'https://github.com/movingpictures83/MakeCliques',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(291,	'MakeScaleFree',	99,	'Make a Scale-Free Network (Albert and Barabasi, 2002)',	'https://github.com/movingpictures83/MakeScaleFree',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(292,	'MakeSynthetic',	99,	'Make a Synthetic Network',	'https://github.com/movingpictures83/MakeSynthetic',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(293,	'MakeSynthetic2',	99,	'Make a Synthetic Network (more complex)',	'https://github.com/movingpictures83/MakeSynthetic2',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(294,	'Microarray',	99,	'Process microarray data, generate gene expression levels',	'https://github.com/movingpictures83/Microarray',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(295,	'MIMOSA',	99,	'Data fitting through MIMOSA (Finak et al, 2013)',	'https://github.com/movingpictures83/MIMOSA',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(296,	'MINT',	99,	'Multivariate INTegration (Rohart et al, 2017)',	'https://github.com/movingpictures83/MINT',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(297,	'NorTA',	99,	'Normal-To-Anything algorithm',	'https://github.com/movingpictures83/NorTA',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(298,	'PC',	99,	'PC algorithm for causal structure (Spirtes et al, 2000)',	'https://github.com/movingpictures83/PC',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(299,	'PCMCI',	99,	'PCMCI algorithm for time-series data (Runge et al, 2019)',	'https://github.com/movingpictures83/PCMCI',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(300,	'PCStable',	99,	'PC-Stable algorithm for causal structure (Colombo and Matthuis, 2014)',	'https://github.com/movingpictures83/PCStable',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(301,	'QueryPathways',	99,	'Obtain all metabolic pathways involving a single or pair of taxa/metabolites.',	'https://github.com/movingpictures83/QueryPathways',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(302,	'Reactome',	99,	'Query the Reactome database (Fabregat et al, 2017)',	'https://github.com/movingpictures83/Reactome',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(303,	'STRING',	99,	'Query the STRING database (Szklarczyk et al, 2011)',	'https://github.com/movingpictures83/STRING',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(304,	'SVARS',	99,	'Calculate counterfactuals using SVARS (Kilian et al, 1998)',	'https://github.com/movingpictures83/SVARS',	4,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(305,	'WattsStrogatz',	99,	'Generate synthetic network (Watts and Strogatz, 1998)',	'https://github.com/movingpictures83/WattsStrogatz',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836'),
(306,	'WikiPathway',	99,	'Query the WikiPathway database (Pico et al, 2008)',	'https://github.com/movingpictures83/WikiPathway',	1,	'2020-09-13 18:51:08.18836',	'2020-09-13 18:51:08.18836');

-- 2020-11-03 20:22:08.287493+00
